src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -12,35 +13,63 @@
 import seedu.address.model.person.Person;
 
 /**
- * Deletes a person identified using it's displayed index from the address book.
+ * Deletes a person identified using its displayed index or name from the address book.
  */
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Deletes the person identified by the index number or the full name used in the displayed person list.\n"
```

Comment:
this line is longer than 120 characterss so it does not pass the java CI.

Reply:
modified, all checks have passed now